public/app/plugins/panel/canvas/components/connections/ConnectionSVG.tsx

```diff
@@ -200,7 +220,16 @@ export const ConnectionSVG = ({
                   stroke={strokeColor}
                   strokeWidth={strokeWidth}
                   fill={'none'}
-                  markerEnd={`url(#${CONNECTION_HEAD_ID})`}
+                  markerEnd={
+                    arrowDirection === ConnectionDirection.Forward || arrowDirection === ConnectionDirection.Both
+                      ? `url(#${CONNECTION_HEAD_ID_END})`
+                      : undefined
```

Comment:
maybe we can extract the logic for `markerEnd`/`markerStart` since we're reusing it?  😁

Reply:
Good suggestion!